components/o3-form/src/css/components/error-summary.css

```
@@ -0,0 +1,34 @@
+.o3-form__error-summary {
```

Does this have a `.o3-form` parent element? If not:

```suggestion
.o3-form-error-summary {
```

---

maybe? or I would think it might have `<form>` element that has a class of `o3-form`? or do you think this error summary won't end up inside the form element? o2 form element has classes [this way](https://o2.origami.ft.com/?path=/docs/o2-core_components-o-forms-readme--docs#error-summary)

---

Enforcing that it's a child of `o3-form` would make its placement less flexible – which may or may not be what we want from a design perspective – but I think you're probably right that it is. Perhaps run in by Deyan to decide. Do we want to enforce the error summary is displayed within the form or might it be displayed at the top of the page, with some other elements, followed be the form?

---

we agreed that currently this will be part of the Form element

---

Have we aligned on the convention? We use a mix of declaring a field as a BEM element, and declaring it as a BEM block.